dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about",
```

Comment:
This aliasing surprised me a bit.

Reply:
This is what i meant by 'not wrong, but very generic'. I was trying to constrain myself to json that read well to non-linked-data people e.g. by having evocative field names & keeping the structure simple, but i wasn't able to find ontology terms out there that accurately describe the relationships between these entities in the biological imaging context. I was able to find lots of terms within domains e.g. describing protocols & methods, but struggled to find terms connecting them. I suspect foundingGide work might provide a solution there, and we could then switch over the ontological term without upsetting the json at a later date? But keen to hear more about the constraints you think are important for the metadata,

Reply:
> not wrong, but very generic

*nods*

> I suspect foundingGide work might provide a solution there

:+1:

> But keen to hear more about the constraints you think are important for the metadata,

Perhaps what might help would be a collection from challengers (or really, anyone) on the terms they would like to use. I imagine the best we can hope for at the moment would be a UNION of various sources. (I've listed on the [agenda](https://hackmd.io/3emKqKQsT_2U35vLepzDEQ?edit) for the 17th)

Reply:
As an alternative to retain better interoperability, you could also handle this by having rules in a profile that specify what @type (and other properties) the entity referenced by `about` should have (e.g. maybe it must be a BioChemEntity). But that is more helpful for devs than non-linked-data folks, I guess